x-pack/plugins/security_solution/public/common/lib/telemetry/constants.ts

```diff
@@ -77,6 +77,11 @@ export enum TelemetryEventTypes {
   OnboardingHubStepOpen = 'Onboarding Hub Step Open',
   OnboardingHubStepFinished = 'Onboarding Hub Step Finished',
   OnboardingHubStepLinkClicked = 'Onboarding Hub Step Link Clicked',
+  ManualRuleRunOpenModal = 'Manual Rule Run Open Modal',
+  ManualRuleRunExecute = 'Manual Rule Run Execute',
+  ManualRuleRunCancelJob = 'Manual Rule Cancel Job',
+  EventLogFilterByRunType = 'Event Log Filter By Run Type',
+  EventLogShowSourceEventDateRange = 'Event Log Show Source Event Date Range',
```

Comment:
These are just strings, right? Is it possible/preferable to add some more structure, here? There are a lot of descriptive words here and it's hard to untangle exactly what's being expressed.

```suggestion
  EventLogShowSourceEventDateRange = 'Event Log -> Show Source Event -> Date Range',
```

Reply:
I was suggesting that we add some kind of hierarchical delimiter to _all_ of these descriptions, not just the one 😅 . I'd value consistency over improving a single item, here, so IMO I'd revert this change.

Reply:
Stepping in here cause it took me some time to understand what `'Event Log -> Show Source Event -> Date Range'` meant. But if I understand correctly, I clicking on the `Show Source Event Date Range` on the Execution Results tab:

![image](https://github.com/elastic/kibana/assets/5354282/fb4716a6-ecc5-4b6e-aece-f483155ffc33)

So yes, I would either:
- revert to the version without the arrows: `'Event Log Show Source Event Date Range'`
- delimit only `Event log` from the rest. Something like: `'Event Log -> Show Source Event Date Range'` or `'Event Log | Show Source Event Date Range'`

Maybe even adding the suffix `Toggle Switch` at the end: `'Event Log | Show Source Event Date Range Toggle Switch`

Reply:
And the same for `EventLogFilterByRunType`